vm-migration/src/lib.rs

```diff
@@ -305,6 +305,9 @@ pub trait Migratable: Send + Pausable + Snapshottable + Transportable {
         Ok(MemoryRangeTable::default())
     }
 
+    fn start_migration(&mut self) {
```

Comment:
```suggestion
    fn start_migration(&mut self) -> std::result::Result<(), MigratableError> {
```

Reply:
Make sure this return a `Result`, which means that your default implementation will return `Ok(())`.